### PR TITLE
Feature review

### PR DESCRIPTION
**Pull request by jaitken**
added review component and added getFamilyId() to shamazin service

### CODE REVIEW

File: src/components/ItemPage/Shamazin.jsx [hunk 1]
```diff
@@ -24,14 +26,21 @@ class Shamazin extends React.Component{
...
         ShamazinService.getItemById(this.state.itemID)
             .then(response=>{
-                this.setState({item:response.data})
-            })
+                this.setState({item:response.data});
+            });
+            
+            ShamazinService.getFamilyId(this.state.itemID)
+            .then( response=>{
+                console.log("FamilyId(Shamazin): "+ response.data);
```
Comment (mattfreshwaters) [inline src/components/ItemPage/Shamazin.jsx]: Will we need console logs for final product? Or were you just using this for manual testing purposes

**Review by mattfreshwaters — APPROVED** [review]
lgtm with 1 comment

File: src/components/ItemPage/Shamazin.jsx (hunk 1 above)
Comment (jaitken) [reply src/components/ItemPage/Shamazin.jsx]: Dang I tried to get rid of all of them, just missed this one I'll make sure to remove it

**Review by FeiYGH — APPROVED** [review]
looks good

**Pull request merged**